genai_processors/contrib/tests/genai_langchain_streaming_test.py

```diff
@@ -0,0 +1,126 @@
+import unittest
+from absl.testing import parameterized
+
+from genai_processors import content_api, processor
+from genai_processors.contrib.genai_langchain_streaming import GenAILangChainProcessor
+
+
+class DummyChunk:
+    """A tiny stand-in for a LangChain chunk with just a `.content` field."""
+    def __init__(self, content: str):
+        self.content = content
+
+
+class DummyChatModel:
```

Comment:
Nit: can you call it `MockChatModel` or `FakeChatModel`. We have a guideline to avoid terms with negative connotations in the codebase and "Dummy" is one of them.